tests/test_common/test_utils/test_checkpoints.py

```diff
@@ -0,0 +1,37 @@
+import os
+import unittest
+
+import numpy as np
+import torch
+import torch.optim as optim
+import torchvision.models as models
+
+from code_soup.common.utils.checkpoints import Checkpoints
+
+
+class TestCheckpoints(unittest.TestCase):
+    def test_save(self):
+        """
+        Test that the model is saved
+        """
+        model_save = models.resnet18(pretrained=True)
```

Comment:
choose a smaller model like discriminator from models.gan, this makes the process a lot slower, and save the files in ./input because that folder is ignored during tests and the files are not committed.